sapbtp-operator-charts/templates/deployment.yml

```diff
@@ -12,6 +12,8 @@ spec:
   selector:
     matchLabels:
       control-plane: controller-manager
+      app.kubernetes.io/instance: sap-btp-operator
+      app.kubernetes.io/name: sap-btp-operator
```

Comment:
What are these labels used for?

Reply:
thanks for catching this! my apologies, this should be templated.

Reply:
these already exist as labels for the pod
https://github.com/SAP/sap-btp-service-operator/blob/e166d789a6b6f10fe71e8ce11281a95e89d2dae1/sapbtp-operator-charts/templates/deployment.yml#L30-L31

there was just discrepancy between `labels` and `matchLabels`